website/docs/r/ssm_association.html.markdown

```diff
@@ -60,7 +60,10 @@ DOC
 
 resource "aws_ssm_association" "foo" {
   name        = "test_document_association-%s"
-  instance_id = "${aws_instance.foo.id}"
+  targets {
```

Comment:
the schemaVersion above is using 1.2 - can we update that to 2.0 to match the ssm_association?

Reply:
I went ahead and just removed the resource declarations above and instead provide an example `${aws_ssm_document.example.name}` reference so we don't need to keep two documentation pages in sync. The details present before were not necessary here.